Cargo.toml

```diff
@@ -19,17 +19,18 @@ derive-new = { version = "0.6.0", default-features = false }
 derive_more = { version = "1", default-features = false }
 log = { default-features = false, version = "0.4.22" }
 
+fnv = "1"
```

Comment:
Should we use the hash function from std? Also we might want to add a link to the original type hash repo in a Notice file similar to how Burn handles that: https://github.com/tracel-ai/burn/blob/main/NOTICES.md

Reply:
I'll add the notice, as for the hasher, there's two problems.
One, the default hasher isn't stable and isn't meant to be - it's meant to prevent key collisions and that involves random seeds.
The other problem is `no_std`. Rust's `DefaultHasher` is only available in `std`, not `core` or `alloc`. So we need `fnv` for no_std compatibility.